### PR TITLE
[R5U_1799] move to GitHub actions

### PR DESCRIPTION
**Pull request by saturnflyer**
Drop Travis, move to GitHub Actions.  Add a `bundle_all` rake task

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: repo-checks
+on:
+  push:
+    branches:
+      - '**' # run on push to any branch, but not pushes to tags
+  pull_request:
```
Comment (grosser) [inline .github/workflows/actions.yml:6]: prefer reusing our regular config ... unless this is temporary for the initial PR

**Review by grosser — APPROVED** [review]

**Comment by grosser** [comment]
... remove travis config ?

File: .github/workflows/actions.yml (hunk 1 above)
Comment (zenspider) [reply .github/workflows/actions.yml:6]: Please point us. We fake it every time. Cribbed the above from classic so that we could get branches under test.

**Review by zenspider — APPROVED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (saturnflyer) [reply .github/workflows/actions.yml:6]: Yeah. Not sure where to find the regular config

**Review by zenspider — CHANGES_REQUESTED** [review]
derp. yeah. remove travis config while you're in here.

**Review by zenspider — APPROVED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (grosser) [reply .github/workflows/actions.yml:6]: https://github.com/zendesk/stronger_parameters/blob/master/.github/workflows/actions.yml#L1-L6

**Pull request merged**